java/partitions/splitterv2/src/main/java/sleeper/splitterv2/core/split/FindPartitionSplitPoint.java

```diff
@@ -110,13 +110,12 @@ public interface SketchesLoader {
         Sketches load(String filename) throws IOException;
     }
 
-    public static SketchesLoader loadSketchesFromFile(Schema schema, Configuration conf) {
-        return (filename) -> new SketchesSerDeToS3(schema).loadFromHadoopFS(new Path(filename), conf);
+    public static SketchesLoader loadSketchesFromFile(Schema schema) {
+        return (filename) -> new S3SketchesStore(S3Client.create(), S3TransferManager.create()).loadFileSketches(filename, schema);
     }
 
-    public static SketchesLoader loadSketchesFromFile(TableProperties tableProperties, Configuration conf) {
-        SketchesSerDeToS3 serDe = new SketchesSerDeToS3(tableProperties.getSchema());
-        return (filename) -> serDe.loadFromHadoopFS(new Path(filename), conf);
+    public static SketchesLoader loadSketchesFromFile(TableProperties tableProperties) {
+        return (filename) -> new S3SketchesStore(S3Client.create(), S3TransferManager.create()).loadFileSketches(filename, tableProperties.getSchema());
```

Comment:
This will create a new S3 client and transfer manager every time we load sketches for a file. Please ensure these can be reused. They can be passed in as parameters to this method, like the Hadoop configuration was before.